src/SingleReview.jsx

```diff
@@ -0,0 +1,94 @@
+import {useParams} from 'react-router-dom'
+import {useState,useEffect} from 'react'
+import { getSingleReview } from './api'
+import { formatDate } from './api'
+import AddReviewLikes from './AddReviewLikes'
+
+function SingleReview() {
+    const [singleReview,setSingleReview] = useState({})
+    const {review_id} = useParams();
```

Comment:
Great use of Params to ensure a user can access the individual review by going directly to the URL